config/manager/manager.yaml

```diff
@@ -16,6 +16,7 @@ spec:
       labels:
         control-plane: capz-controller-manager
         aadpodidbinding: capz-controller-aadpodidentity-selector
```

Comment:
If there is no plan to support pod identity anymore after workload identity is supported, maybe add a `TODO` or an issue to track removing this label.